BrainPortal/app/views/bourreaux/show.html.erb

```diff
@@ -262,6 +263,33 @@
           <div class="field_explanation">Optional. Careful, this is inserted as-is in the command-line for submitting jobs.</div>
         <% end %>
 
+        <% if @bourreau.cms_class == "ScirAmazon" %>
+	  <% t.edit_cell(:amazon_ec2_access_key_id, :header => "Amazon EC2 Access Key Id") do %>
+            <%= text_field_tag "bourreau[amazon_ec2_access_key_id]", @bourreau.amazon_ec2_access_key_id, :size => 60 %><br/>
+            <div class="field_explanation">See <a href="http://docs.aws.amazon.com/AWSSdkDocsRuby/latest/DeveloperGuide/ruby-dg-setup.html">Amazon SDK documentation</a>.</div>
```

Comment:
add target=_blank so that clicking on documentation link doesn't force the user to the page...